tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -433,6 +433,13 @@ spec:
 
 			stdOut := helper.CmdShouldPass("odo", "link", "EtcdCluster/example")
 			Expect(stdOut).To(ContainSubstring("Successfully created link between component"))
+			stdOut = helper.CmdShouldFail("odo", "link", "EtcdCluster/example")
+			Expect(stdOut).To(ContainSubstring("already linked with the service"))
+
+			stdOut = helper.CmdShouldPass("odo", "unlink", "EtcdCluster/example")
```

Comment:
can we check if the secrets were actually removed? or if the SBR was deleted?

Reply:
I tried to do this by adding `oc get sbr` in the test. But it takes some time for the SBR to disappear from the output. I'll have to add a few seconds' timeout to be able to validate this.